test/e2e_node/e2e_remote.go

```diff
@@ -188,6 +188,7 @@ func RunRemote(archive string, host string, cleanup bool, junitFileNumber int) (
 	glog.Infof("Copying test artifacts from %s", host)
 	scpErr := getTestArtifacts(host, tmp)
 	exitOk := true
+	glog.Errorf("vishh: %v", scpErr)
```

Comment:
Should this be printed inside the "if scpErr != nil" below? Looks like a temporary/local change